Lib/test/test_time.py

```diff
@@ -1041,6 +1041,36 @@ def test_object_to_timespec(self):
             with self.assertRaises(ValueError):
                 pytime_object_to_timespec(float('nan'), time_rnd)
 
+@unittest.skipUnless(sys.platform == "darwin", "test weak linking on macOS")
+class TestTimeWeaklinking(unittest.TestCase):
+    # These test cases verify that weak linking support on macOS works
```

Comment:
Please add a bpo number (where I would expect a longer explanation of what "weak linking" is ;-)).

Reply:
Would a reference to the Mac readme be acceptable?  I'd still have to write the explanation, but that would keep the explanation in the source tree instead of in an issue that won't be updated once this is merged.

Reply:
There is now a reference to Mac/README.txt where added some text about weak linking.